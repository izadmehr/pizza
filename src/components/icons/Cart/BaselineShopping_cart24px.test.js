import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

import Image from "./Cart-icon";

describe("BaselineShopping_cart24px.svg generated styled component", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Image />);
  });

  it("renders correctly according to snapshot", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it("has dimensions greater than zero", () => {
    const dimensions = Image.getDimensions();
    expect(dimensions.width).not.toBe("0");
    expect(parseInt(dimensions.width, 10)).toBeGreaterThan(0);
    expect(dimensions.height).not.toBe("0");
    expect(parseInt(dimensions.height, 10)).toBeGreaterThan(0);
  });
});
