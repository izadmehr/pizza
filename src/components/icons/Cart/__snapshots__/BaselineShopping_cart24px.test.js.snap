// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BaselineShopping_cart24px.svg generated styled component renders correctly according to snapshot 1`] = `
<StyledComponent
  forwardedClass={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          "
  ",
          [Function],
          "
",
        ],
      },
      "defaultProps": Object {
        "children": Array [
          <path
            d="M7 18c-1.1 0-1.99.9-1.99 2S5.9 22 7 22s2-.9 2-2-.9-2-2-2zM1 2v2h2l3.6 7.59-1.35 2.45c-.16.28-.25.61-.25.96 0 1.1.9 2 2 2h12v-2H7.42c-.14 0-.25-.11-.25-.25l.03-.12.9-1.63h7.45c.75 0 1.41-.41 1.75-1.03l3.58-6.49A1.003 1.003 0 0 0 20 4H5.21l-.94-2H1zm16 16c-1.1 0-1.99.9-1.99 2s.89 2 1.99 2 2-.9 2-2-.9-2-2-2z"
            fill="white"
          />,
        ],
        "viewBox": "0 0 24 24",
      },
      "displayName": "BaselineShopping_cart24px",
      "getDimensions": [Function],
      "getDimensionsCss": [Function],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "svg",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
  viewBox="0 0 24 24"
>
  <path
    d="M7 18c-1.1 0-1.99.9-1.99 2S5.9 22 7 22s2-.9 2-2-.9-2-2-2zM1 2v2h2l3.6 7.59-1.35 2.45c-.16.28-.25.61-.25.96 0 1.1.9 2 2 2h12v-2H7.42c-.14 0-.25-.11-.25-.25l.03-.12.9-1.63h7.45c.75 0 1.41-.41 1.75-1.03l3.58-6.49A1.003 1.003 0 0 0 20 4H5.21l-.94-2H1zm16 16c-1.1 0-1.99.9-1.99 2s.89 2 1.99 2 2-.9 2-2-.9-2-2-2z"
    fill="white"
    key="key-0"
  />
</StyledComponent>
`;
